api/src/main/java/com/datatorrent/api/DAG.java

```diff
@@ -236,11 +236,22 @@
   public abstract <T> void setAttribute(Attribute<T> key, T value);
 
   /**
-   * <p>setAttribute.</p>
+   *
+   * Use {@link #setOperatorAttribute} instead
    */
```

Comment:
Addressed all the review comments. Travis builds fails randomly. I just updated JavaDoc, so it shouldn't have updated the Travis.